src/efs_parser/BaseFile.py

```diff
@@ -153,13 +171,16 @@ def get_bytes(self, offset, size):
         Returns:
             bytes from offset to (offset + size)
         """
-        if self.local:
+        if self.file_source == "local":
             f = open(self.file, "rb")
             f.seek(offset)
             bin_value = f.read(size)
             f.close()
             return bin_value
-        else:
+        elif self.file_source == "s3":
+            response = self.get_bytes_from_s3(offset,size)
+            return response
```

Comment:
this needs to also be present in Lines 132-133

Reply:
The lines from 115 - 154 are not used anywhere in the code and I suggest removing them

Reply:
they are used in a different package 'efs_distributed'

Reply:
Done